source/portable/NetworkInterface/esp32/NetworkInterface.c

```diff
@@ -152,6 +220,8 @@ esp_err_t wlanif_input( void * netif,
     {
         /* Set the packet size, in case a larger buffer was returned. */
         pxNetworkBuffer->xDataLength = len;
+        pxNetworkBuffer->pxInterface = pxMyInterface;
+        pxNetworkBuffer->pxEndPoint = FreeRTOS_MatchingEndpoint( pxInterface, pcBuffer );
```

Comment:
Here in 2 lines you're using `pxMyInterface` as well as `pxInterface`. It looks like only the former is available here?

Reply:
Done, thanks